src/test/Fake.Dotnet.Cli.IntegrationTests/TemplateTests.fs

```diff
@@ -1,11 +1,75 @@
-﻿module Fake.Dotnet.Cli.IntegrationTests.TemplateTests
+﻿module Fake.DotNet.Cli.IntegrationTests.TemplateTests
```

Comment:
Please also rename the folder in git, otherwise there will be problems (which is a bit unfortunate to do on windows as you have to temporarily rename to something else)